Tools/scripts/pretty_diff_size.py

```diff
@@ -20,7 +20,7 @@
 def sizes_for_file(filepath):
     """Get binary size information with size."""
     print("Get binary size of %s" % filepath)
-    cmd = "size %s" % (filepath,)
+    cmd = "arm-none-eabi-size %s" % (filepath,)
```

Comment:
is this mandatory ? it restricts the tool to just arm-none-eabi build.

Reply:
You are correct. So for the waf_build_summary tool we use this version for arm boards so I was trying to standardize, but did a poor job. 

This isn't a good idea for other types of boards. And obvious by the massive number of failing tests because crash_log wont appear in sitl or linux boards....

Reply:
Switched back. I'm not sure the differences but for the future we may want to detect the type of build and switch like the waf script does.